### ISSUE TITLE
[BUG] DDEX shows zero balance even though there is enough

### ISSUE THREAD

**Issue by mifeng** [CoinAlpha/hummingbot#642]
**Describe the bug**
Users have reported issues with our DDEX connector such as not being able to place orders due to insufficient balance, even though their wallets have enough. This may be due to the fact that DDEX geo-fences certain regions. I replicated this issue, but didn't find the root cause.

**Config used for both tests**
```
maker_market: ddex
maker_market_symbol: WETH-DAI

# size of each order
order_amount: 0.1

# Time in seconds before cancelling and rerunning strategy
# If tick size is 60, the bot cancels active orders and reruns after a minute
cancel_order_wait_time: 60

# How far away from mid price to place the bid order
# Expressed in decimals : 0.01 = 1% away from mid price at that time
# Example if mid price is 100 and bid_place threshold is 0.01
# Your bid is placed at 99
bid_place_threshold: 0.002

# How far away from mid price to place the ask order
# Expressed in decimals : 0.01 = 1% away from mid price at that time
# Example if mid price is 100 and ask_place threshold is 0.01
# Your bid is placed at 101
ask_place_threshold: 0.002

mode: single
```

**Test 1 (no VPN, US region)**
Screenshot shows warnings about insufficient balance but wallet has balance and the bot places orders:
<img width="703" alt="Screen Shot 2019-08-08 at 1 29 44 PM" src="https://user-images.githubusercontent.com/1795030/62737096-e0199b00-b9e3-11e9-88f3-7cbc1f3581a3.png">

Logs:
[ddex balance errors.log](https://github.com/CoinAlpha/hummingbot/files/3483631/ddex.balance.errors.log)

**Test 1 (VPN, United Kingdom region)**
Screenshot shows zero wallet balance but this is the same wallet as above. No orders are placed:
<img width="703" alt="Screen Shot 2019-08-08 at 1 44 20 PM" src="https://user-images.githubusercontent.com/1795030/62737205-16571a80-b9e4-11e9-9c0f-e233af012eee.png">

Logs:
[ddex errors 2.log](https://github.com/CoinAlpha/hummingbot/files/3483637/ddex.errors.2.log)

**Comment by dennisocana** [comment]
I'm using VM in HK/SG region. Unable to place orders yet but wallet balance is showing.

![image](https://user-images.githubusercontent.com/50150287/62739824-77c0cd00-ba68-11e9-8516-76b07bec1bfd.png)

**Comment by chowryan** [comment]
@dennisocana you might just not have enough $ for minimum order size.

**Comment by dennisocana** [comment]
Added some more tokens in wallet. Test bot is placing order.

![image](https://user-images.githubusercontent.com/50150287/62740874-94123900-ba6b-11e9-9542-682d5f0fdc0e.png)

**Comment by veecos2** [comment]
I am able to place orders with a VPN in UK region on an infura node. Infact pureMM is able to cancel orders and place them as per cancel order wait time.

**Comment by veecos2** [comment]
Two minor issues I have noticed are 1. its unable to fetch status updates on the order which it places which appears in logs (should be fixable) 2. it takes time for the available balance to get updated (after the orders are cancelled) and in the meanwhile the strategy gives a warning that there are is not enough balance which gets rectified as soon as it gets the actual update, which is not that long. I was able to run it without any breaking issue every 60 seconds.
